### ISSUE TITLE
[Design] 10-10EZ Household Section - final Sketch file

### ISSUE THREAD

**Issue by jessicastump** [department-of-veterans-affairs/va.gov-team#56616]
## Issue Description
Update the Sketch file with a QA page that includes all the final user flow and content decisions made so it aligns with the end staging output. 

---
## Tasks
- [x] Update the wireflow in the [QA Sketch page](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/p/12B7255E-72F6-4C6A-83C3-81BF9A1CEBDE/canvas)
- [x] Add use cases to wireflow for reference
- [x] Update the design file with final decisions on content and FE 
- [x] Design files for review page / open and edit states
- [x] Update all mobile and desktop designs with final content in the QA Sketch page

MOVE TO NEW TICKET - Post-Production source of truth clean up

- [ ] Once live, update the general wireflows Sketch page source of truth
- [ ] Once live, update the page in Sketch file 4. Household information source of truth

## Acceptance Criteria
- [x] All design files are updated to reflect what will be used during Staging Review
---

**Comment by jessicastump** [comment]
![wireflow-56616](https://user-images.githubusercontent.com/98367393/234108709-81e071a8-29dd-4b5a-ab58-0693d0aba36c.jpg)
- [Sketch link to image](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/v/vkJ2P1/a/9lP93nx)

**Comment by jessicastump** [comment]
Review pages by [use case](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Household%20section%20redesign/Staging%20Review_QA/Household%20Section%20Optimization%20Use%20Cases.md#10-10-ez-household-section-optimization-use-cases)

- [Use Case A](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/r99yJkK)
- [Use Case B](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/0AAoZxG)
- [Use Case C](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/OeeqzE4)
- [Use Case D](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/r99y3MP)
- [Use Case E](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/P113qej)
- [Use Case F](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/mPPvVQk)
- [Use Case G](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ZVVb8ov)
- [Use Case H](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/zyyRwnl)
- [Use Case I](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/5nnV5aE)

**Comment by jessicastump** [comment]
After the bi-weekly content meeting on 4/28, we are still waiting on content review for the income and deductible pages. 

All screens/content in the QA file up to those last two pages have been reviewed.

**Comment by jessicastump** [comment]
Opened the following ticket for follow-up work and closing this ticket.

- Updating source of truth files once this work goes live - #58021
